tests/basic.test.tsx

```diff
@@ -459,7 +465,7 @@ it('can subscribe to the store', () => {
   expect(listener).toHaveBeenCalledTimes(1)
   expect(listener).toHaveBeenCalledWith(1)
```

Comment:
I am surprised this test passed, but maybe I am seeing something wrong (@dai-shi)

Reply:
Ah, nice catch. Could you change L457 to `listener(s.value, prevValue)` please?

Reply:
Done 🐻